cdap-ui/app/cdap/components/RulesEngineHome/RulesEngineServiceControl/index.js

```diff
@@ -89,24 +150,31 @@ export default class RulesEngineServiceControl extends Component {
             {T.translate(`${PREFIX}.benefits.title`)}
 
             <ul>
-              <li>{T.translate(`${PREFIX}.benefits.b1`)}</li>
-              <li>{T.translate(`${PREFIX}.benefits.b2`)}</li>
-              <li>{T.translate(`${PREFIX}.benefits.b3`)}</li>
-              <li>{T.translate(`${PREFIX}.benefits.b4`)}</li>
+              <li>
+                <span className="fa fa-laptop" />
```

Comment:
We have `icon-laptop` in the `symbol-defs.svg`, so you can do `IconSVG` here too

Reply:
Talked @tonybach the shaded pixel ratio for this icon seemed off when using `IconSVG` so defaulted to font awesome. Need to investigate why this happens.